.github/workflows/release-please.yaml

```diff
@@ -21,24 +21,47 @@ jobs:
       all: ${{ toJSON(steps.release.outputs) }}
       releases_created: ${{ steps.release.outputs.releases_created }}
       paths_released: ${{ steps.release.outputs.paths_released }}
-  sbom:
+  artifacts:
     needs: release-please
     runs-on: ubuntu-latest
     if: ${{ needs.release-please.outputs.releases_created }}
     strategy:
       matrix:
         release: ${{ fromJSON(needs.release-please.outputs.paths_released) }}
+    env:
+      TAG: ${{ fromJSON(needs.release-please.outputs.all)[format('{0}--tag_name', matrix.release)] }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+      # Bundle SBOM
       - name: Generate SBOM
         uses: CycloneDX/gh-gomod-generate-sbom@v1
         with:
           version: v1
           args: mod -licenses -json -output bom.json ${{ matrix.release }}
       - name: Upload SBOM to release
-        uses: svenstaro/upload-release-action@v2
+        uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844
```

Comment:
why changing the action?

Reply:
Good point, I switched to `softprops/action-gh-release` as I have seen the usage in other repositories I maintain. But, seems `svenstaro/upload-release-action` fits well here. 

I will made the swithch

Reply:
One weakness of `svenstaro/upload-release-action` is that it lacks multi-file upload capability. I will refactor the code to use `svenstaro/upload-release-action` in a single job step